src/templates/invoice/types.ts

```diff
@@ -22,16 +22,16 @@ export interface Invoice extends Document {
   billFrom?: CompanyInfo;
   billTo?: BillingAddress;
   billableItems?: InvoiceItem[];
-  subtotal?: string | number; //to cater for: SGD $2 or $2
-  tax?: string | number; //to cater for: GST - 7% or 7
-  taxTotal?: string | number; //to cater for: SGD $2 or $2
-  total?: string | number; //to cater for: SGD $2 or $2
+  subtotal?: string; //to cater for: SGD $2 or $2
+  tax?: string; //to cater for: GST - 7% or 7
+  taxTotal?: string; //to cater for: SGD $2 or $2
+  total?: string; //to cater for: SGD $2 or $2
```

Comment:
think u can remove the comments already, since react-jsonschema form does not support multiple options for type so far i think